pkgs/development/ocaml-modules/topkg/default.nix

```diff
@@ -4,6 +4,10 @@ if !stdenv.lib.versionAtLeast ocaml.version "4.01"
 then throw "topkg is not available for OCaml ${ocaml.version}"
 else
 
+let
+  run = "ocaml -I ${findlib}/lib/ocaml/${ocaml.version}/site-lib/ pkg/pkg.ml";
```

Comment:
You may want to leave a comment that `run` and `buildPhase` are part of the interface for use by other Ocaml packages and should be changed with care.

Reply:
Done.

Reply:
Thanks!